pkg/workspace/apply_test.go

```diff
@@ -1001,6 +1001,55 @@ func TestApply_IsolatedWorkspaces(t *testing.T) {
 				ReadOnly:  true,
 			}},
 		},
+	}, {
+		name: "add workspaces to taskspec",
+		ts: v1beta1.TaskSpec{
+			Sidecars: []v1beta1.Sidecar{{
+				Name: "conflicting volume mount sidecar",
+				VolumeMounts: []corev1.VolumeMount{{
+					Name:      "mount-path-conflicts",
```

Comment:
it looks like this test case is for conflicting mount paths, can you rename to reflect what's being tested here?

Reply:
No it wasn't. Its my mistake. Copy and pasting the above test and then modifying it lead to this confusion. I apologize. I will update it to better reflect the test its actually doing.